libraries/aws/shadow/cbmc/proofs/Shadow_MatchTopic/Shadow_MatchTopic_harness.c

```diff
@@ -28,22 +28,25 @@
 #include "shadow_cbmc_state.h"
 
 #ifndef TOPIC_STRING_LENGTH_MAX
-    #define TOPIC_STRING_LENGTH_MAX    128U
+    #define TOPIC_STRING_LENGTH_MAX    256U
 #endif
 
 void harness()
 {
-    ShadowMessageType_t messageType = ShadowMessageTypeMaxNum;
+    ShadowMessageType_t messageType;
     const char * pTopicName; 
     uint16_t topicNameLength;
-    const char * pThingName = NULL;
-    uint16_t thingNameLength = 0U;
+    const char ** pThingName;
+    uint16_t thingNameLength;
+
     __CPROVER_assume( topicNameLength < TOPIC_STRING_LENGTH_MAX );
+    pThingName = mallocCanFail( sizeof( char * ) );
```

Comment:
```suggestion
    pThingName = mallocCanFail( sizeof( char * ) );
    pThingNameLength = mallocCanFail( sizeof( uint16_t ) );
```

Reply:
will modify it.

Reply:
please review this [commit](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1134/commits/cf59eec1603cd09c71e33bf1514318149b702d44)!